modules/testing-utils/testing-utils/src/main/java/org/motechproject/testing/utils/BasePkgTest.java

```diff
@@ -79,7 +79,7 @@ public void cleanUp() {
     public abstract String getChrootDirProp();
 
     /**
-     * Reads the script with the given name from the classpath and installs it into the temp directory.
+     * Reads the script with the given name from the classpath and installs it into the jdo directory.
```

Comment:
It had to change when I did some auto refactoring with IntelliJ and I missed it. Will fix it in next commit. The same applies to all strange "temp" to "jdo" changes.